tensorhive/app/web/dev/src/components/views/reserve_resources/MySchedule.vue

```diff
@@ -172,6 +171,12 @@ export default {
   watch: {
     parsedNodes () {
       this.fillTable()
+      for (var nodeIndex in this.parsedNodes) {
+        var nodeName = this.parsedNodes[nodeIndex].nodeName
+        this.checkIfAllResourcesSelected(nodeName)
+      }
+      this.checkIfAllNodesSelected()
+      this.loadResources()
```

Comment:
Here loadResources will be emitted. It is required to update the calendar on page reload (more specifically on changing the parseNodes argument).

Reply:
Seems to work, but not per user basis - the info is stored within the browser, so all users using the same browser will see the same list (just like other things in tensorhive actually).

Reply:
Yep, that's how we intended it. Thanks for checking it out.